### PR TITLE
[7.0][IMP] Better modularity and naming convention for account_journal_period_close

### PR DESCRIPTION
**Pull request by adrienpeiffer**
Improve modularity and naming convention for account_journal_period_close

### CODE REVIEW

File: account_journal_period_close/model/account_period.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+from openerp.osv import orm, fields
+
+
+class account_period(orm.Model):
```
Comment (pedrobaeza) [inline account_journal_period_close/model/account_period.py]: Better to use CamelCase

File: account_journal_period_close/model/account_journal_period.py [hunk 2]
```diff
@@ -31,14 +31,6 @@
...
 
-class account_period(orm.Model):
-    _inherit = 'account.period'
-    _columns = {
-        'journal_period_ids': fields.one2many('account.journal.period',
-                                              'period_id', 'Journal states'),
-    }
-
-
 class account_journal_period(orm.Model):
```
Comment (pedrobaeza) [inline account_journal_period_close/model/account_journal_period.py]: Better to use CamelCase (AccountJournalPeriod)

**Comment by pedrobaeza** [comment]
Thanks for the changes.

**Comment by omar7r** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by Olivier-LAURENT** [comment]
:+1:

**Comment by nbessi** [comment]
:+1:

**Pull request merged**